[Cocoa] Create a top-level folder in WebKit for WebKitSwift files

aestes opened the pull request:
#### 1ad215e3864768c9a5c65b5c38c1deb21fae9bed
<pre>
[Cocoa] Create a top-level folder in WebKit for WebKitSwift files
<a href="https://bugs.webkit.org/show_bug.cgi?id=268301">https://bugs.webkit.org/show_bug.cgi?id=268301</a>
<a href="https://rdar.apple.com/121858909">rdar://121858909</a>

Reviewed by Ben Nham.

Moved GroupSession.swift and MarketplaceKitWrapper.swift from WebKit/UIProcess/Cocoa to WebKit/WebKitSwift.
Sorted the top-level file and folder references in the WebKit project.

* Source/WebKit/WebKit.xcodeproj/project.pbxproj:
* Source/WebKit/WebKitSwift/GroupActivities/GroupSession.swift: Renamed from Source/WebKit/UIProcess/Cocoa/GroupActivities/WKGroupSession.swift.
* Source/WebKit/WebKitSwift/MarketplaceKit/MarketplaceKitWrapper.swift: Renamed from Source/WebKit/UIProcess/Cocoa/MarketplaceKitWrapper.swift.

Canonical link: <a href="https://commits.webkit.org/273695@main">https://commits.webkit.org/273695@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/876c5a3692d0a09bcdc5e992f1823d02ca43716f

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/36359 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/15313 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/38578 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/39076 "Built successfully") | [  ~~🛠 wincairo~~](https://ews-build.webkit.org/#/builders/32/builds/32674 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/17757 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/12344 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/31331 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/36919 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/12926 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/32235 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/11332 "Passed tests") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/42/builds/11356 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/40320 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/32991 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/32814 "Passed tests") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/37289 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/11567 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/50/builds/9446 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/35385 "Passed tests") | 
| | [  ~~🛠 tv-sim~~](https://ews-build.webkit.org/#/builders/46/builds/13277 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/8244 "Built successfully and passed tests") | [  ~~🛠 watch~~](https://ews-build.webkit.org/#/builders/43/builds/12019 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/12427 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

bnham reviewed (approved):
LGTM, please rebase after [this PR merge](https://github.com/WebKit/WebKit/pull/23390/commits/dcebef8667bf352eebdbb94df9fd2f36796ec5f3) because there's definitely a conflict due to renaming `WKApplicationUtilities.swift` to `MarketplaceKitWrapper.swift`.

aestes committed:
[Cocoa] Create a top-level folder in WebKit for WebKitSwift files
https://bugs.webkit.org/show_bug.cgi?id=268301
rdar://121858909

Reviewed by Ben Nham.

Moved GroupSession.swift and MarketplaceKitWrapper.swift from WebKit/UIProcess/Cocoa to WebKit/WebKitSwift.
Sorted the top-level file and folder references in the WebKit project.

* Source/WebKit/WebKit.xcodeproj/project.pbxproj:
* Source/WebKit/WebKitSwift/GroupActivities/GroupSession.swift: Renamed from Source/WebKit/UIProcess/Cocoa/GroupActivities/WKGroupSession.swift.
* Source/WebKit/WebKitSwift/MarketplaceKit/MarketplaceKitWrapper.swift: Renamed from Source/WebKit/UIProcess/Cocoa/MarketplaceKitWrapper.swift.

Canonical link: https://commits.webkit.org/273695@main
